### PR TITLE
Create readme and linters

### PR DESCRIPTION
**Pull request by udeaghad**
create linters, html and css file for review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (mahbubul14) [inline .gitignore:1]: - [ ] Kindly add `node_modules` in the `.gitignore` file.

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+
+
+## Authors
+
+👤 **DOZIE UDEAGHA**
+
+- GitHub: @udeaghad
+- Twitter: @theodoz
+- LinkedIn: [LinkedIn]
```
Comment (mahbubul14) [inline README.md]: - [ ] Kindly add your social links for this part. Here is an example of how to do it linkable:

`- GitHub: [@githubhandle](https://github.com/githubhandle)`

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @udeaghad ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense and the first letter should be capitalized. `Ex: "Fix bug" instead of "fix bug or fixed bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 2 above)
Comment (udeaghad) [reply README.md]: link address updated appropriately

File: .gitignore (hunk 1 above)
Comment (udeaghad) [reply .gitignore:1]: @mahbubul14, node_modules added in the .gitignore file.

**Review by codepantha — APPROVED** [review]
Hi @udeaghad 👋,

## STATUS: APPROVED ✔️  

Good job making the requested changes 👏 👏 
Your project is now complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**